crypto/fipsmodule/rand/urandom_test.cc

```diff
@@ -498,7 +498,7 @@ static std::vector<Event> TestFunctionPRNGModel(unsigned flags) {
   const size_t kPersonalizationStringLength = CTR_DRBG_ENTROPY_LEN;
   const size_t kPassiveEntropyWithWhitenFactor = PASSIVE_ENTROPY_LOAD_LENGTH;
   const bool kHaveRdrand = have_hw_rng_x86_64();
-  const bool kHaveFastRdrand = have_hw_rng_x86_64_fast();
+  const bool kHaveFastRdrand = have_hw_rng_x86_64();
```

Comment:
We should just remove `kHaveFastRdrand`.

Reply:
Ahh correct. Will do when re-writing the tests.